.github/workflows/deploy.yml

```
@@ -0,0 +1,78 @@
+name:  Spring Boot & Gradle CI/CD
+
+on:
+  push:
+    branches: [ develop ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)